pkg/client/fwd_pipe.go

```diff
@@ -19,7 +19,7 @@ type FwdPipeConfig struct {
 	Cookie         uint64
 }
 
-func (c *Client) SetFwdPipeFromBytes(ctx context.Context, binBytes, p4infoBytes []byte, cookie uint64) (*FwdPipeConfig, error) {
+func (c *Client) ChangeFwdPipeFromBytes(ctx context.Context, binBytes, p4infoBytes []byte, cookie uint64, action p4_v1.SetForwardingPipelineConfigRequest_Action) (*FwdPipeConfig, error) {
```

Comment:
I don't like the name here. I think we should stick with the "Set" terminology since this is the name of the RPC. What about `SetFwdPipeFromBytesWithAction`? I don't mind a longer name :)